### DISCUSSION TITLE
Math newline with default MathJax not working

### DISCUSSION THREAD

**Discussion by retodomax** [quarto-dev/quarto-cli#4916] [Q&A] [answered]
I'm trying to switch from RMarkdown to Quarto and something which gives me some problem is to create newlines in display math. Reprex:

```
---
title: Math newline with Quarto
format:
  html
---

# Equation

$$
x = 1 \\
y = 2
$$
```
Seems to be a [problem with Mathjax](https://github.com/mathjax/MathJax/issues/2312) but why does it work in RMarkdown? Maybe another version of MathJax was used? Is is possible to select the same version of MathJax used in RMarkdown?

**Comment by cderv** [comment, marked as answer]
R Markdown ecosystem use Mathjax 2 by default. Not yet upgrade to Mathjax 3 
See discussion at https://github.com/rstudio/bookdown/issues/1364#issuecomment-1225461584 

Default Mathjax in Quarto will be 3 - which unfortunately has this limitation. 

But as explained in the documentation, you can change to what you want: 
https://quarto.org/docs/output-formats/html-basics.html#latex-equations

````markdown
---
title: Math newline with Quarto
format: 
  html:
    html-math-method:
      method: mathjax
      url: "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML"
---

# Equation

$$
x = 1 \\
y = 2
$$
````

You can use other math method too - see the doc

**Reply by cderv** [reply]
I published this as a an example for later reference to it https://examples.quarto.pub/changing-mathjax/

**Reply by retodomax** [reply]
Thanks! Will Quarto's default Mathjax version be upgradend to [version 4](https://docs.mathjax.org/en/latest/upgrading/whats-new-4.0.html) at some point? That would be great : )

**Reply by cderv** [reply]
yes we'll probably try, but it will depend on the possible breaking change, and how pandoc's supports it. 

Though you could use mathjax 4 today by setting the url as shown. So you can use it already without any change on default. (change in default value is always tricky for user's experience and breaking change if any)
